lxd/seccomp/seccomp.go

```diff
@@ -744,30 +737,14 @@ func seccompGetPolicyContent(s *state.State, c Instance) (string, error) {
 	// Policy header
 	policy := seccompHeader
 	allowlist := config["security.syscalls.allow"]
-	if allowlist == "" {
-		allowlist = config["security.syscalls.whitelist"]
-	}
 
 	if allowlist != "" {
-		if s.OS.LXCFeatures["seccomp_allow_deny_syntax"] {
```

Comment:
We should still be checking the underlying liblxc supports the functionality even if our min requirements mean it will have that support.

Reply:
Oh I suppose they will get a liblxc error if not supported, ok then thats fine then.

Reply:
Although the liblxc error may not be very useful, probably better to detect support and return a clear error here instead.